### PR TITLE
Feat autosell

### PR DESCRIPTION
**Pull request by Journeytojah**
Added autosell flag and delay. Feature is still experimental and your transactions and not guaranteed to execute.

Use with caution!

### CODE REVIEW

File: buy.ts [hunk 1]
```diff
@@ -296,6 +308,89 @@ async function buy(
...
+        const latestBlockhash = await solanaConnection.getLatestBlockhash({
+          commitment: commitment,
+        });
+        const messageV0 = new TransactionMessage({
+          payerKey: wallet.publicKey,
+          recentBlockhash: latestBlockhash.blockhash,
+          instructions: [
+            ComputeBudgetProgram.setComputeUnitLimit({ units: 400000 }),
+            ComputeBudgetProgram.setComputeUnitPrice({ microLamports: 200000 }),
+            createAssociatedTokenAccountIdempotentInstruction(
```
[the remark this answers is not in the document]
Comment (Journeytojah) [reply buy.ts]: Didn't figure out how to close the token account, cleaned up the rest.

**Comment by wildyrando** [comment]
it is possible to add feature auto sell by percentase example 15%

File: buy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (fdundjer) [reply buy.ts]: No problem, I'll push the changes :)

**Review by fdundjer — APPROVED** [review]

**Comment by fdundjer** [comment]
> it is possible to add feature auto sell by percentase example 15%

Yes, but out of scope for this PR. Can you file a new issue?

**Pull request merged**